semantic-conventions/src/tests/data/markdown/ref_extends/http.yaml

```diff
@@ -0,0 +1,24 @@
+groups:
+  - id: http.client.attributes
+    type: attribute_group
+    brief: 'This document defines semantic conventions for HTTP client attributes.'
+    attributes:
+      - ref: server.address
+        requirement_level: required
+        brief: Server component of Host header.
```

Comment:
Should we also add a `note` here to make sure that is also working as expected? Also the requirement level message I guess.